### PR TITLE
PV2-634 improve e2e act1 test stop gap

### PR DESCRIPTION
**Pull request by kuzinator**

### CODE REVIEW

File: src/SFA.DAS.Payments.FundingSource.Application/Services/ContractType1RequiredPaymentEventFundingSourceService.cs [hunk 1]
```diff
@@ -82,6 +91,9 @@ public async Task<ReadOnlyCollection<FundingSourcePaymentEvent>> GetFundedPaymen
                 await requiredPaymentsCache.Clear(key).ConfigureAwait(false);
             }
 
+            var funds = string.Join(", ", fundingSourceEvents.GroupBy(f => f.FundingSourceType).Select(f => f.Key.ToString() + ": " + string.Join("+", f.Select(ff => ff.AmountDue.ToString("#,##0.##")))));
```
Comment (samuel-dev-dba) [inline src/SFA.DAS.Payments.FundingSource.Application/Services/ContractType1RequiredPaymentEventFundingSourceService.cs]: Need to show this code some love, please refactor

File: src/SFA.DAS.Payments.FundingSource.LevyFundedProxyService/Handlers/ApprenticeshipContractType1RequiredPaymentEventHandler.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public class ApprenticeshipContractType1RequiredPaymentEventHandler : IHandleMes
 
         public async Task Handle(ApprenticeshipContractType1RequiredPaymentEvent message, IMessageHandlerContext context)
         {
-            paymentLogger.LogInfo($"Processing LevyFundedProxyService event. Message Id : {context.MessageId}");
+            paymentLogger.LogInfo($"Processing ApprenticeshipContractType1RequiredPaymentEvent event. Message Id: {context.MessageId}, Job: {message.JobId}, UKPRN: {message.Ukprn}");
```
Comment (samuel-dev-dba) [inline src/SFA.DAS.Payments.FundingSource.LevyFundedProxyService/Handlers/ApprenticeshipContractType1RequiredPaymentEventHandler.cs]: I believe we agreed to remove these multiple levels of try catch

File: src/SFA.DAS.Payments.FundingSource.Application/Services/ContractType1RequiredPaymentEventFundingSourceService.cs (hunk 1 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.FundingSource.Application/Services/ContractType1RequiredPaymentEventFundingSourceService.cs]: done

File: src/SFA.DAS.Payments.FundingSource.LevyFundedProxyService/Handlers/ApprenticeshipContractType1RequiredPaymentEventHandler.cs (hunk 2 above)
Comment (kuzinator) [reply src/SFA.DAS.Payments.FundingSource.LevyFundedProxyService/Handlers/ApprenticeshipContractType1RequiredPaymentEventHandler.cs]: removed one layer

**Pull request merged**